app/src/main/java/net/programmierecke/radiodroid2/data/DataRadioStation.java

```diff
@@ -190,4 +207,55 @@ public JSONObject toJson(){
 
 		return null;
 	}
+
+	public interface ShortcutReadyListener {
```

Comment:
I'm inclined to have these two classes outside of `DataRadioStation`.

Reply:
Yes, a shortcut manager class and, even more important, a station icon manager class would make sense.